mod/account/lib/card/mod/account.rb

```diff
@@ -1,4 +1,4 @@
 Cardio::Railtie.config.tap do |config|
   config.account_password_length = 8
-  config.account_password_requirements = %i[lower upper special_char number]
+  config.account_password_requirements = %i[lower upper special_char number letter]
```

Comment:
I'm not sure we need both "letter" _and_ "lower"/"upper".  I think in this context we just drop letter.